pkg/syncer/metadatasyncer.go

```diff
@@ -90,7 +91,16 @@ var (
 	// isMultiVCenterFssEnabled is true if the Multi VC support FSS is enabled, false otherwise.
 	isMultiVCenterFssEnabled bool
 	// nodeMgr stores the manager to interact with nodeVMs.
-	nodeMgr node.Manager
+	nodeMgr                  node.Manager
+	ResourceAPIgroupSnapshot = "snapshot.storage.k8s.io"
```

Comment:
Can you comment out this for now and add a NOTE?  Support for snapshot quota will be added at a later time.

Reply:
done, removed snapshot related code for now, will add it later when support for snapshot quota gets enabled